modules/examples/src/main/java/com/ibm/cloud/platform_services/global_search/v2/GlobalSearchExamples.java

```diff
@@ -49,33 +49,42 @@ public static void main(String[] args) throws Exception {
 
     try {
       String searchCursor = null;
+      System.out.println("search() result:");
 
       // begin-search
+
       SearchOptions searchOptions = new SearchOptions.Builder()
         .query("GST-sdk-*")
-        .fields(new java.util.ArrayList<String>(java.util.Arrays.asList("*")))
+        .fields(new java.util.ArrayList<>(java.util.Arrays.asList("*")))
```

Comment:
```suggestion
        .addFields("*")
```
For options fields that are lists, we generate an "addXXX()" method to add an element to the list for that field.   I think using these "addXXX()" methods in the examples makes it easier for users.